cms/templates/content_libraries/xblock_iframe.html

```diff
@@ -156,7 +156,7 @@
 <!-- The default stylesheet will set the body min-height to 100% (a common strategy to allow for background
   images to fill the viewport), but this has the undesireable side-effect of causing an infinite loop via the onResize
   event listeners below, in certain situations.  Resetting it to the default "auto" skirts the problem.-->
-<body style="min-height: auto">
+<body style="min-height: auto; background-color: white;">
```

Comment:
Do you think using transparent here will make it more adaptable to mfe theming?

Reply:
I think a lot of the XBlocks and course content are designed assuming that there is a white background. This is also used for LTI launches, and then we can't control what background color it appears on. So I think white is the safest for now. But I'm not sure.